examples/writablepipe.js

```diff
@@ -0,0 +1,15 @@
+var nano = require('..')
+var pub   = nano.socket('pub'),       push  = nano.socket('push');
+var sub   = nano.socket('sub'),       pull  = nano.socket('pull');
+pub.bind('tcp://127.0.0.1:3333');     push.bind('tcp://127.0.0.1:4444');
+sub.connect('tcp://127.0.0.1:3333');  pull.connect('tcp://127.0.0.1:4444');
+
+sub.setEncoding('utf8'); //sub socket will get utf8 strings instead of Buffers
```

Comment:
This should be documented in the readme. Something like `this can be used to always convert received messages to Strings of an explicit encoding` or something.